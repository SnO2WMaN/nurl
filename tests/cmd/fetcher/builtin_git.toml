args = [
    "https://github.com/nix-community/nurl",
    "refs/tags/v0.3.0",
    "--fetcher",
    "builtins.fetchGit",
]
