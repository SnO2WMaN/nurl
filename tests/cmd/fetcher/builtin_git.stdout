builtins.fetchGit {
  url = "https://github.com/nix-community/nurl";
  ref = "refs/tags/v0.3.0";
}